packages/ui/src/components/filters/CheckboxFilter/CheckboxFilter.tsx

```diff
@@ -168,12 +169,14 @@ const CheckboxFilter = ({
     };
 
     useEffect(() => {
-        const newFilters = bumpCheckedFilterFirst()
+        const initFiltered = bumpCheckedFilterFirst();
+        const newFilters = (initFiltered.length ? initFiltered : filters)
             .filter((f) => !isEmpty(f.data))
```

Comment:
Je ferais une seule boucle:
```suggestion
            .filter(({ data, name }) => {
                if(isEmpty(f.data)) return;
                ...
            }
```